models/evc.py

```diff
@@ -374,6 +379,15 @@ def as_dict(self):
         evc_dict["flow_removed_at"] = self.flow_removed_at
         evc_dict["updated_at"] = self.updated_at
 
+        if keys:
+            selected = {}
+            for key in keys:
+                if key == "enable":
```

Comment:
@Alopalao why did this key needed to be handled differently? 

I haven't spotted the case where `"enable"` is being passed.

Reply:
Found them in the [e2e tests](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/7de86a1445019360c39a7be415d49d1c7cd18555/tests/test_e2e_10_mef_eline.py#L453) and also we are accepting it as an update [option](https://github.com/kytos-ng/mef_eline/blob/46397d678748b7c2e182bdaf33a4e5cc6876d046/models/evc.py#L211C6-L211C6)

Reply:
I see. Alright. Let's keep this then, following the same pattern.